rel-eng/README.md

```diff
@@ -0,0 +1,59 @@
+# Release Howto
+
+Our release process is documented using Jupyter notebook (interractive documentation).
+
+To read the docs view the `gem_release.ipynb` on GitHub. To run the notebook Jupyter is needed. Jupyter is web based interractive python
+environment that can mix executable snippets with markdown.
+
+The markdown parts should inform the releaser about what is going to be done and 
+the executable pieces should do the actual work. If anything goes unexpected it is easy
+ to stop, investigate, fix and re-run the step.
+ 
```

Comment:
Somewhere here we should also specify possible dependencies.

When I tested I hitted the following requirements:
 - transifex-client
 - python-slugify
 - asciidoc
 - To make non-interactive step 18 I needed to configure credentials for rubygems.org (I've used api key)